### PR TITLE
Clean up Some Clippy Warnings

### PR DESCRIPTION
**Pull request by Slowki**
This PR cleans up some Clippy warnings

### CODE REVIEW

File: crates/backend/src/ast.rs [hunk 1]
```diff
@@ -39,7 +39,7 @@ pub struct Import {
 
 #[cfg_attr(feature = "extra-traits", derive(Debug, PartialEq, Eq))]
 pub enum ImportKind {
-    Function(ImportFunction),
+    Function(Box<ImportFunction>),
```
Comment (alexcrichton) [inline crates/backend/src/ast.rs]: We aren't really going for speed here, could this be left as is to keep it consistent ?

File: crates/backend/src/codegen.rs [hunk 2]
```diff
@@ -31,19 +31,19 @@ fn to_ident_name(s: &str) -> Cow<str> {
 impl ToTokens for ast::Program {
     // Generate wrappers for all the items that we've found
     fn to_tokens(&self, tokens: &mut TokenStream) {
-        for export in self.exports.iter() {
```
Comment (alexcrichton) [inline crates/backend/src/codegen.rs]: I personally prefer the previous way this was written, could it stay as it was?

**Comment by alexcrichton** [comment]
Thanks!

File: crates/backend/src/ast.rs (hunk 1 above)
Comment (Slowki) [reply crates/backend/src/ast.rs]: :+1:

**Pull request merged**